File: backend/app/schemas/vinculo_unidade.py
from pydantic import BaseModel
from datetime import datetime

class VinculoUnidadeBase(BaseModel):
    unidade_externa: str
    unidade_medida_id: int

class VinculoUnidadeCriar(VinculoUnidadeBase):
    pass

class VinculoUnidadeSchema(VinculoUnidadeBase):
    id: int
    criado_em: datetime
    atualizado_em: datetime

    class Config:
        from_attributes = True

File: frontend/src/types/vinculoUnidade.ts
export interface VinculoUnidade {
  id: number;
  unidade_externa: string;
  unidade_medida_id: number;
  criado_em?: string;
  atualizado_em?: string;
}

export interface VinculoUnidadeCriar {
  unidade_externa: string;
  unidade_medida_id: number;
}